pandas/core/window.py

```diff
@@ -910,8 +910,9 @@ def skew(self, **kwargs):
 
     Returns
     -------
-    Series or DataFrame (matches input)
-        Like-indexed object containing the result of function application
+    Series or DataFrame
+        Returned object type is dictated by the caller of the %(name)s
```

Comment:
As a non-native speaker, I found your original "determined" better than "dictated" (but maybe not more correct :-)). Or simpler "Returned object is of the same type as the caller ..." ?